src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/ua/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MoyoLogo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
```

Comment:
Use semantic tags where possible. Instead of using `<ul>` directly within `<nav>`, consider wrapping the list in a `<nav>` tag to define a navigation section.